### PR TITLE
[DE] Add `Play_style`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Notes for reviewers

- **line 108**: Maybe `verzeichnen` is a good alternative to `registrieren`?
- **line 116**: If you have a better translation for `catcher` and `dashing`/`to dash`, please suggest it!
- **line 148**: In case there is a well-written German article for mechanical keyboards, please mention it, though the article must have the same content as the english one.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Play_style/de.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+### Andere Stile
+
+- **Gitarren-/Schlagzeug-Set** (5/6K): Der Standardcontroller für das Gitarren-bezogene ([Guitar Hero](https://de.wikipedia.org/wiki/Guitar_Hero))/Schlagzeug-bezogene ([DrumMania](https://en.wikipedia.org/wiki/GuitarFreaks_and_DrumMania)) Rhythmusspiel. Spieler streamen auf ihrer Gitarre oder schlagen auf ihrem Schlagzeug und Zimbeln, um Befehle einzugeben. Schlagzeuger haben einen Nachteil im Vergleich zu Gitarrenspieler, weil sie ohne ein spezielles Fußpedal nur zwei Eingaben gleichzeitig machen können.
+- **Para Para Controller** (5K): Das ist der Standardcontroller für die Serie [Para Para Paradise](https://wikipedia.org/wiki/Para_Para_Paradise). Spieler bewegen ihre Arme, Ellbogen und Hände direkt über den Abstandssensoren, um Eingaben zu machen, basierend auf der Position der Hand und des Abstandssensors. Der Nachteil ist die Sensitivität des Eingabeempfängers und die physische Limitierung des Spielers.
```
Comment (respektive) [inline wiki/Play_style/de.md:173]:
```suggestion
- **Para Para Controller** (5K): Das ist der Standardcontroller für die Serie [Para Para Paradise](https://wikipedia.org/wiki/Para_Para_Paradise). Spieler bewegen ihre Arme, Ellbogen und Hände direkt über den Abstandssensoren, um Eingaben zu machen, basierend auf der Position der Hand und die Abstandssensors. Der Nachteil ist die Sensitivität des Eingabeempfängers und die physische Limitierung des Spielers.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Play_style/de.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+- **Gamepad**: Ein Gamepad ist eine mögliche Spielweise in bei den verschiedenen Konsolenvarianten der Serie Taiko no Tatsujin (der Taiko Drum Master mit eingeschlossen). Spieler registrieren Don/Katu durch das Drücken der Front (rechter Don oder rechtes Rot), des Pfeils (linker Don oder linkes Rot) und der oberen Buttons (linkes/rechtes Katu oder linkes/rechtes Blau).
+
+## ![](/wiki/shared/mode/catch.png) osu!catch
+
+### Tastatur
+
+Eine Tastatur ist die meist verwendete, standardmäßige Spielweise für osu!catch.
+
+Um mit einer Tastatur zu spielen, drücken Spieler die linken und rechten Pfeiltasten, um den Catcher zu bewegen und halten `linksShift` gedrückt, um zu dashen.
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Play_style/de.md:116]: I thought so too 👍

File: wiki/Play_style/de.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+### Tanzmatte
+
+Tanzmatten sind Controller für Spiele wie die Serien [Dance Dance Revolution](https://de.wikipedia.org/wiki/Dance_Dance_Revolution) (DDR) und [Pump It Up](https://en.wikipedia.org/wiki/Pump_It_Up_(video_game_series)) (PIU). ([StepMania](https://wikipedia.org/wiki/StepMania) ist ein halbwegs bekannter, von der Community betriebener Klon dieser beiden Spiele).
+
+Beim Spielen mit Tanzmatten "tanzen" Spieler auf der Matte, um Befehle einzugeben. DDR-Pads haben 4 Felder: hoch, runter, links und rechts. DDR-Solo-Pads fügen oben-links und oben-rechts hinzu. PIU-Pads have 5 Felder: oben-links, oben-rechts, unten-links, unten-rechts und in der Mitte. Es gibt auch einige 9 Pads, die sowohl mit PIU als auch mit DDR kompatibel sind, ohne dass ein Pad ausgetauscht werden muss. Normale DDR- und PIU-Pads werden oft im Doppelmodus in ihrem jeweiligen Spiel verwendet, wobei ein Spieler 2 Felder für einen Song benutzt. DDR-Solo-Spiele sind alle im Einzelspielermodus, daher sind sie in der Regel nicht verdoppelt.
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Play_style/de.md:164]: Is it really `die`? I thought it is a different version of `Es gibt ein Feld in der Mitte`.

---

Is `Pads mit 9 Feldern` better?

File: wiki/Play_style/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Play_style/de.md:173]: What is the grammatical reasoning behind this change? Do you want me to write this:

> basierend auf der Position der Hand und den Abstandssensoren

File: wiki/Play_style/de.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (respektive) [reply wiki/Play_style/de.md:164]: Oh i meant `und die Mitte`. Your suggestions sounds better yeah, I edited my comment.

File: wiki/Play_style/de.md (hunk 1 above)
Comment (respektive) [reply wiki/Play_style/de.md:173]: My bad, I must have misread that sentence. It's correct as is, ignore this :skull:

Comment (The-Last-Cookie) [reply wiki/Play_style/de.md:173]: No worries. I wanted to make sure that I understand you correctly 😄

**Review by respektive — APPROVED** [review]
lgtm

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**